Jasmine/Common/ViewControllers/HomeScreenViewController.swift

```diff
@@ -4,6 +4,9 @@ class HomeScreenViewController: UIViewController {
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         if let gridGame = segue.destination as? GridGameViewController {
             gridGame.segueWith(GridViewModel())
+
```

Comment:
just to be visually clear the two code blocks, but will rewrite this class again next time.